.gitignore

```diff
@@ -22,3 +22,5 @@ lm_eval/caching/.cache
 # don't track files created by wandb
 wandb
 examples/wandb
+results/*
```

Comment:
Let's remove this change from .gitignore.

Reply:
Done